### PR TITLE
Feature/groceries list

### PR DESCRIPTION
**Pull request by Geno22**
Added data structure for the shopping list and a simple implementation on the MainScreen.

### CODE REVIEW

File: app/src/main/res/layout/activity_main_screen.xml [hunk 1]
```diff
@@ -87,6 +87,18 @@
         app:layout_constraintTop_toBottomOf="@+id/house_imageButton"
         app:layout_constraintVertical_bias="0.064" />
 
+    <Button
+        android:id="@+id/liste_view_change2"
```
Comment (Arkadyus) [inline app/src/main/res/layout/activity_main_screen.xml:91]: list_view_change is probably cleaner unless it conflicts with any other id

**Review by Arkadyus — APPROVED** [review]
Everything looks clean, moving ShopItem inside of ShopList can be refactored later as it is not that big of an issue and is quickly fixed.

File: app/src/main/java/com/github/houseorganizer/houseorganizer/ShopListAdapter.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+import android.view.ViewGroup;
+import android.widget.CheckBox;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+public class ShopListAdapter extends RecyclerView.Adapter<ShopListAdapter.ItemsHolder> {
+
+    public class ItemsHolder extends RecyclerView.ViewHolder{
+        public CheckBox checkBox;
```
Comment (aindreias) [inline app/src/main/java/com/github/houseorganizer/houseorganizer/ShopListAdapter.java:15]: Great idea to use a `CheckBox` button! I will probably switch the simple task buttons to this as well.

**Review by aindreias — APPROVED** [review]
I like the idea of switching between task and grocery lists. Overall good job and thanks for adding unit tests.

**Pull request merged**